README.md

```diff
@@ -362,6 +362,20 @@ Make sure lockfile version is higher than v1 (Node.js 15 or higher).
 
 See [docs/npm.md](docs/npm.md) for more details.
 
+### yarn
+
+<https://v3.yarnpkg.com/>
+
+Current version: v3
+
+Cachi2 supports Yarn by parsing [package.json](https://v3.yarnpkg.com/configuration/manifest/)
+file present in the source repository and downloading the declared dependencies.
+
+If not checked in to the repository, a lockfile (``yarn.lock``) will automatically be generated by
```

Comment:
I did a few tests, and the prefetch fails when the yarn.lock file is not present:

```
Internal Error: sample-nodejs-app@workspace:.: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile
```

Maybe we should wrap this into a nicer error? In any case, I think to keep the current ethos of Cachi2, we shouldn't allow a repo without a lockfile to be processed.

Reply:
Reworded the docs accordingly, but a fix will have to be posted for the issue.

Reply:
> Maybe we should wrap this into a nicer error? In any case, I think to keep the current ethos of Cachi2, we shouldn't allow a repo without a lockfile to be processed.

Opened https://github.com/containerbuildsystem/cachi2/pull/440 to take care of that.